sqlite/src/parse.y

```diff
@@ -1566,8 +1575,11 @@ uniqueflag(A) ::= .        {A = OE_None;}
   ){
     ExprList *p = sqlite3ExprListAppend(pParse, pPrior, 0);
 #if defined(SQLITE_BUILDING_FOR_COMDB2)
-    /* Allow sort order in FK index columns. */
-    if( hasCollate && pParse->db->init.busy==0)
+    /* Allow sort order in FK index columns. Views do not allow sort order.
```

Comment:
Check why we need this diff from upstream.

Reply:
Upstream sqlite touches this code while parsing column lists in FK, CTE & VEIW definitions, where it disallows sort-order altogether.

In Comdb2, unlike upstream, we allow sort orders in FK index column lists, but not in VIEWs and CTE - thus the additional check. (still need to fix this for CTE tough, TODO).

Reply:
@akshatsikarwar : As discussed, I have remove the support for use of sort order in DDL FK defs.